tools/autograd/gen_variable_type.py

```diff
@@ -692,7 +728,14 @@ def emit_view_lambda(unpacked_bindings: List[Binding]) -> str:
             else:
                 updated_unpacked_args.append(arg)
 
-        replay_view_call = emit_dispatch_call(f, input_base, updated_unpacked_args)
+        operator_name = f.func.name.name
+        overload_name = f.func.name.overload_name
+
+        dispatcher_sig = DispatcherSignature.from_schema(f.func)
+        dispatcher_exprs = dispatcher_sig.exprs()
+        type_signature = f"{dispatcher_sig.returns_type()} ({', '.join([a.type.cpp_type() for a in dispatcher_exprs])})"
```

Comment:
Are these actually needed here?

Reply:
This was a relic of the amount of interactive rebasing that I did in this diff stack 😛  Definitely not, I'll take it out